ext/queue_tasks/Civi/QueueHelper.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+namespace Civi;
+
+use Civi\Core\Exception\DBQueryException;
+
+/**
+ * Queue helper.
+ *
+ * This comes from https://gist.github.com/totten/fa830cfced9bb7a92dea485f5422055a
+ * & will hopefully be moved to core.
+ */
+class QueueHelper {
```

Comment:
This class is from the second commit - ie the second line of thinking (it's the only thing in that commit)